### PR TITLE
1.6.0 blog post and package.json update

### PR DESCRIPTION
**Pull request by SergioCrisostomo**
- 1.6.0 release post
 - package.json update

### CODE REVIEW

**Comment by ptlakt448** [comment]
<a href="http://www.Techstole.com">Techstole</a>
i am enjoy and learn more
i like it and keep it up.good job
thanks
<a href="http://www.Techstole.com">techstole.com</a>

File: blog/posts/mootools-1-6-0-release.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#1.6.0 is out!
```
Comment (arian) [inline blog/posts/mootools-1-6-0-release.md]: Not the jekyll like header?

For example https://raw.githubusercontent.com/mootools/website/master/blog/posts/a-better-way-to-use-elements.md

has 

```
---
title: "A Better Way to use Elements"
date: "Fri, 19 Mar 2010 22:41:41 +0000"
author: "Sean McArthur"
tags: "all"
permalink: "2010/03/19/a-better-way-to-use-elements/"
---
```

**Comment by SergioCrisostomo** [comment]
@arian thank you, yes, it should be there. Fixed.
Had lost it in the last import from the Gist.

**Comment by ptlakt448** [comment]
<a href="http://www.songs1.in">songspk</a>
<a href="http://www.songs1.in">songs.pk</a>
<a href="http://www.songs1.in">songs pk</a>
i am enjoy and learn more
i like it and keep it up.good job
thanks

**Comment by ptlakt448** [comment]
<a href="http://www.songspk49.com">songspk</a>
i am enjoy and learn more
i like it and keep it up.good job
thanks

**Pull request merged**